MovieGallery/Pages/Index.cshtml

```diff
@@ -12,5 +12,5 @@
         <h1 class="text-center" style="background-color: darkred; color: yellow; "> Movie Gallery</h1>
     </div>
 
-    <img src="\images\movieGallery.JPG" width="100%" height="400" alt="movieGallery" />
+    <img src="\images\movieGallery.JPG" style="width: 100%" height="400" alt="movieGallery" />
```

Comment:
Removed the warning on Index.cshtml as img width= '100%' is not a valid value of attribute width' by replacing with equivalent style code - style="width: 100%"
https://stackoverflow.com/questions/49677911/why-x-is-not-a-valid-value-of-attribute-width-in-visual-studio